R/score_untrained.R

```diff
@@ -83,9 +86,19 @@ score_untrained <- function(single_task_data, oracle_output_data, model_id_list,
       value = NA, importance = NA
     )
     # bind the new rows to df_importance
-    importance_scores <- bind_rows(df_importance, missing_model_rows)
+    combined_df <- bind_rows(df_importance, missing_model_rows)
+    # reorder the columns
+    importance_scores <- data.frame(model_id = model_id_list) |>
+      left_join(
+        combined_df,
+        by = "model_id"
+      ) |>
+      select(-c(.data$output_type_id, .data$value)) |>
```

Comment:
are the `.data$` needed in this line and the one below?

Reply:
I will note that when I ran the tests in R, testthat suggested this change (among many other warnings) as well.

Reply:
e.g. 
```
Warning (test-score_untrained.R:150:5): Assign NAs for missing data with output type: quantile metric: wis
Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
i Please use `"value"` instead of `.data$value`
```